vet.sh

```diff
@@ -157,7 +157,21 @@ grpc.WithTimeout
 http.CloseNotifier
 info.SecurityVersion
 resolver.Backend
-resolver.GRPCLB' "${SC_OUT}"
+resolver.GRPCLB
+proto.RegisterType is deprecated
+proto.RegisterEnum is deprecated
+proto.RegisterFile is deprecated
+proto.FileDescriptor is deprecated
+proto.MessageType is deprecated
+proto.RegisteredExtensions is deprecated
+extDesc.Filename is deprecated
```

Comment:
Sort those lines also? Would also help to find duplicates.

Reply:
Done